package.json

```diff
@@ -4,11 +4,13 @@
   "version": "1.0.0",
   "angular-cli": {},
   "scripts": {
-    "start": "ng serve",
+    "start": "node ./bin/www",
```

Comment:
Would this be considered a breaking change? Both commands spin up a server so the end result is no different.